[ITER 40000] Evaluating test: L1 0.059811333586510855 PSNR 21.881342160074333 [30/09 10:15:57]

[ITER 40000] Evaluating train: L1 0.03897766321897507 PSNR 24.53205261230469 [30/09 10:15:59]

[ITER 40000] Saving Checkpoint [30/09 10:15:59]
Training progress:  12%|█▏        | 59990/499999 [1:20:42<10:49:48, 11.29it/s, Loss=0.6047814, gaussion_pts=775877]The number of merge points is: 82611 [30/09 10:43:34]
the following position in xyz is NaN: (tensor([    0,     0,     0,  ..., 29999, 29999, 29999], device='cuda:0'), tensor([0, 1, 2,  ..., 0, 1, 2], device='cuda:0')) [30/09 10:43:34]
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [48,0,0], thread: [74,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [48,0,0], thread: [75,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [48,0,0], thread: [78,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [48,0,0], thread: [79,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [48,0,0], thread: [82,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [48,0,0], thread: [83,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [48,0,0], thread: [86,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [48,0,0], thread: [87,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [48,0,0], thread: [90,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [48,0,0], thread: [91,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [48,0,0], thread: [94,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [48,0,0], thread: [95,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [16,0,0], thread: [74,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [16,0,0], thread: [75,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [16,0,0], thread: [78,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [16,0,0], thread: [79,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [16,0,0], thread: [82,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [16,0,0], thread: [83,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [16,0,0], thread: [86,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [16,0,0], thread: [87,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [16,0,0], thread: [90,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [16,0,0], thread: [91,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [16,0,0], thread: [94,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [16,0,0], thread: [95,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
Traceback (most recent call last):
  File "/mnt/hdd4T/www_home/.cache/JetBrains/RemoteDev/dist/4e0ec52ff7f75_pycharm-professional-2024.1/plugins/python/helpers/pydev/pydevd.py", line 1535, in _exec
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "/mnt/hdd4T/www_home/.cache/JetBrains/RemoteDev/dist/4e0ec52ff7f75_pycharm-professional-2024.1/plugins/python/helpers/pydev/_pydev_imps/_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "/mnt/hdd4T/www_home/project/learn/gaussian-splatting/merge_test.py", line 352, in <module>
    args.merge_points_num)
  File "/mnt/hdd4T/www_home/project/learn/gaussian-splatting/merge_test.py", line 132, in training
    _ = prepare_merge(gaussians, merge_num=merge_points_num)
  File "/mnt/hdd4T/www_home/project/learn/gaussian-splatting/utils/merge_utils.py", line 40, in prepare_merge
    merge_group_indices = find_merge_group(gaussians, points_query_for_kdtree, merge_group_size=merge_group_size)
  File "/mnt/hdd4T/www_home/project/learn/gaussian-splatting/utils/merge_utils.py", line 76, in find_merge_group
    dists, inds = torch_kdtree.query(points_query, nr_nns_searches=k)
  File "/mnt/hdd4T/www_home/miniconda3/envs/c3dgs/lib/python3.7/site-packages/torch_kdtree/nn_distance.py", line 109, in query
    inds = self.shuffled_ind[result_idx.long()]
RuntimeError: CUDA error: device-side assert triggered
GPUassert: device-side assert triggered /mnt/hdd4T/www_home/project/learn/gaussian-splatting/submodules/torch_kdtree/src/kdtree_g.cu 144
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [2,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [3,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [6,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [7,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [10,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [11,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [14,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [15,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [18,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [19,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [22,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [23,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [26,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [27,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [30,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [31,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [34,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [35,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [38,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [39,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [42,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [43,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [46,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [47,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [50,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [51,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [54,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [55,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [58,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [59,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [62,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.
/opt/conda/conda-bld/pytorch_1659484801627/work/aten/src/ATen/native/cuda/IndexKernel.cu:91: operator(): block: [57,0,0], thread: [63,0,0] Assertion `index >= -sizes[i] && index < sizes[i] && "index out of bounds"` failed.

Process finished with exit code 198
